brion/blueConfig.cpp

```diff
@@ -65,6 +68,12 @@ inline std::string lexical_cast( const brion::BlueConfigSection& b )
 }
 }
 
+namespace
+{
+    const char* const SPIKE_FILE = "/out.dat";
```

Comment:
I moved all the definitions here but I guess you're referring to another place out of this file, possibly a public header, right?